tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1700,6 +1700,16 @@ error: Unexpected return value from void function. Return type annotations are r
    |     ^^^^^^^^^^^^ Unexpected return value from void function. Return type annotations are required for methods.
 
 
+error: Structs must be declared at the top-level of a program: Struct 'SS' is not defined at the top-level
+   --> ../../../examples/tests/invalid/inference.test.w:88:3
+   |  
+88 | /   struct SS {
+89 | |     a: Array<num>;
+90 | |     b: Array<str>;
+91 | |   }
+   | \\\\---^ Structs must be declared at the top-level of a program: Struct 'SS' is not defined at the top-level
```

Comment:
To make sure we're testing the right things, could you adjust https://github.com/winglang/wing/blob/main/examples/tests/invalid/inference.test.w#L88 to remove this error (moving the struct outside the closure)

Reply:
The inference test was updated to remove that error, and new struct tests were added to verify that structs can't be made in preflight or inflight contexts.